osm2streets/src/lib.rs

```diff
@@ -86,21 +87,36 @@ impl StreetNetwork {
         for i in [id.i1, id.i2] {
             self.intersections.get_mut(&i).unwrap().roads.push(id);
             self.sort_roads(i);
+            // Recalculate movements and complexity.
```

Comment:
Since we do this twice and it's non-trivial, could we split out a helper method similar to `sort_roads`? Maybe `fn recalculate_movements`